src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/health/HealthCheckTest.kt

```diff
@@ -101,4 +101,19 @@ class HealthCheckTest : IntegrationTestBase() {
         }
       )
   }
+
```

Comment:
I spotted this missing test this morning, and fixed it before i knew about this change - so I lumped it in with this PR.